recipes/javafx/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "javafx" %}
+{% set version = "11.0.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://gluonhq.com/download/javafx-{{ version }}-sdk-linux  # [unix and x86_64]
+  url: http://gluonhq.com/download/javafx-{{ version }}-sdk-mac  # [osx and x86_64]
+  sha256: 40ef06cd50ea535d45403d9c44e9cb405b631c547734b5b50a6cb7b222293f97  # [unix and x86_64]
+  sha256: e98158812db1a0037cdaf85824adff384e41e3edf046fda145479ce6057cb514  # [osx and x86_64]
+  fn: javafx-{{ version }}-sdk.zip 
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - coreutils >=8
```

Comment:
I think you can remove this ``build`` here, just let the host and run it should be fine

Reply:
done.